tests/test_simple.py

```diff
@@ -53,6 +53,20 @@ def setup(self, device):
         time.sleep(2)
 
 
+class SlowSetupWithCustomHealthLitAPI(SimpleLitAPI):
+    def setup(self, device):
+        self.model = lambda x: x**2
+        time.sleep(2)
+
+    def health(self) -> bool:
+        # Custom health check logic
+        if not hasattr(self, "_start_time"):
+            self._start_time = time.time()
+
+        # Healthy if the server has been running for at least 5 seconds
+        return time.time() - self._start_time >= 5
```

Comment:
```suggestion
        # Health check passes after 5 seconds from the first time it is called. 
        if not hasattr(self, "_start_time"):
            self._start_time = time.time()
        return time.time() - self._start_time >= 5
```

`setup` is called in a subprocess and `health` is called in the main process